SetFlags.cmake

```diff
@@ -62,6 +62,9 @@ macro(set_flags)
 		set(CMAKE_EXE_LINKER_FLAGS_RELEASE    "${CMAKE_EXE_LINKER_FLAGS_RELEASE}    /LTCG")
 		set(CMAKE_SHARED_LINKER_FLAGS_RELEASE "${CMAKE_SHARED_LINKER_FLAGS_RELEASE} /LTCG")
 		set(CMAKE_MODULE_LINKER_FLAGS_RELEASE "${CMAKE_MODULE_LINKER_FLAGS_RELEASE} /LTCG")
+
+		# Make build use Unicode:
+		add_definitions(-DUNICODE -D_UNICODE)
```

Comment:
Well, I don't understand how std::function works either, doesn't mean I shouldn't experiment :)

Am I correct in thinking that, even under UNICODE, strings, both literal and dynamic, will only become wide if prefixed by `L` or given a type `wchar_t`? Because the server ran, printed messages, parsed recipes, etc. perfectly fine.